doc/developer_guide/parsing.rst

```diff
@@ -225,15 +226,15 @@ layer is translated to PSyIR (see issue #753). The current
 implementation could be improved but in practice the lfric code does
 not fall foul of this restriction.
 
-There is also a constraint that invoke arguments can not be
-expressions or functions if they include a variable as it is then
-difficult to determine the datatype of the argument. However,
-arbitrary structures and arrays are supported as are literal
-expressions.
+There is also a constraint that invoke arguments cannot be expressions
+or functions as it is then difficult to determine the datatype of the
+argument. However, arbitrary structures and arrays are supported as
```

Comment:
Comma after "supported" please.

Reply:
I, will, never, know, when, to, add, or, not, to, add.